src/containers/Permissions/index.jsx

```diff
@@ -329,6 +330,7 @@ class Permissions extends React.Component {
 										size="medium"
 										content="Edit mode"
 										onClick={getKeys}
+										disabled={keyWeightWarn}
```

Comment:
if user going to change wif keys they cannot to do that, block save button at permission page